components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/manager/FxaAccountManager.kt

```diff
@@ -487,6 +493,14 @@ open class FxaAccountManager(
         syncStatusObserverRegistry.register(observer, owner, autoPause)
     }
 
+    /**
+     * Unregister a [SyncStatusObserver] from being informed about "sync lifecycle" events.
+     * The method is safe to call even if the provided observer was not registered before.
+     */
+    fun unregisterForSyncEvents(observer: SyncStatusObserver) {
```

Comment:
Didn't realize we didn't have an unregister before. 😄

Reply:
This probably means there could've been other leaks also but hopefully after this they'll be no more.

Reply:
Yeah, and I see others on the AccountFragment in Fenix too, but none of the sync events observer.